var/spack/repos/builtin/packages/gurobi/package.py

```diff
@@ -56,5 +60,7 @@ def install(self, spec, prefix):
 
     @run_after("install")
     def gurobipy(self):
-        with working_dir("linux64"):
-            python("setup.py", "install", "--prefix={0}".format(self.prefix))
+        if self.spec.satisfies("+python"):
```

Comment:
Here you can just:
```python
@run_after("install", when="+python")
```
and leave the method as is

Reply:
I'm not sure what your comment implies about leaving the method as-is, given that you're recommending an edit.  Are you suggesting that line 57 would be changed to include the "+python" logic?

Reply:
Yes, and that no edit are then necessary to the method's body.